### ISSUE TITLE
Show IPv4 and IPv6 nodes separately

### ISSUE THREAD

**Issue by Charlio99** [3bl3gamer/storjnet-info#5]
I don't know if it's possible, but it would be cool to switch between IPv4, IPv6 and both in the map.

**Comment by 3bl3gamer** [comment]
No, sorry, IPv6-nodes now are not tracked at all. But... maybe later I'll find a way to search them.
